src/org/labkey/test/tests/FieldEditorRowSelectionActionTest.java

```diff
@@ -27,10 +26,14 @@
 import java.util.List;
 
 @Category(DailyA.class)
-@BaseWebDriverTest.ClassTimeout(minutes = 2)
+@BaseWebDriverTest.ClassTimeout(minutes = 3)
 public class FieldEditorRowSelectionActionTest extends BaseWebDriverTest
 {
     private final static String PROJECT_NAME = "Field Editor Row Selection Action Test";
+    private final static List<String> expectedHeaders = Arrays.asList("Name", "Range URI", "Required", "Lock Type", "Lookup Container", "Lookup Schema", "Lookup Query",
```

Comment:
Should we include a test in a folder in which the `Ontology` module is not enabled, in order to ensure `"Source Ontology", "Concept Import Column", "Concept Label Column", "Principal Concept Code"` columns accurately do not get rendered, or would that be an overzealous test?

Reply:
Done.